lib/taurus/qt/qtgui/panel/taurusvalue.py

```diff
@@ -1233,19 +1236,21 @@ def setLabelConfig(self, config):
         """
         # backwards compatibility: this method used to work for setting
         # an arbitrary text to the label widget
+        self._labelConfig = config
         try:
```

Comment:
IMHO, setLabelConfig needs to be rewritten. The problems I see:
- It should do what the docstrings says: set a fragment, not an arbitrary formar str
-  the first try-except makes no sense as it is now (unless I am misunderstanding something)...

Perhaps setLabelConfig should be deprecated and let only for the old  purpose with a warning, and only `setText()` / `setText_()` should be used internally

Reply:
`self._labelConfig` is used to keep the current configuration, and selected it in onChangeLabelConfig 
and should be move before the comment.

It was done like this in #496, IMO we could change the docstring to accept format string and we could simplify the check

> Perhaps setLabelConfig should be deprecated and left only for providing bck-compat, and only setText() / setText_() should be used internally. The same for {get,reset}LabelConfig.
Maybe, but I could be a "big" API change

Reply:
I personally would prefer to deprecate {get,set,reset}LabelConfig (to simplify an already messy API)